.github/workflows/build-assets.yml

```diff
@@ -330,8 +330,7 @@ jobs:
           build-args: |
             PKG_FILE=${{ steps.ln-container.outputs.k8s_pkg_file }}
           tags: |
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.internal_tag }}
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.candidate_base_branch }}
+            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/hpccpl-docker-local/hpcc-platform/${{ matrix.os }}/platform-core-ln:${{ needs.preamble.outputs.internal_tag }}
```

Comment:
@gfortil - I think prior to this change the action failed to push because it was trying to push to the directory 'ln':
```
ERROR: failed to solve: failed to push ***/ln/platform-core-ubuntu-22.04:internal_9.4.4-rc1: unexpected status from POST request to https://***/v2/ln/platform-core-ubuntu-22.04/blobs/uploads/: 404 Not Found
Error: buildx failed with: ERROR: failed to solve: failed to push ***/ln/platform-core-ubuntu-22.04:internal_9.4.4-rc1: unexpected status from POST request to https://***/v2/ln/platform-core-ubuntu-22.04/blobs/uploads/: 404 Not Found
```

with this PR change, it will now be trying to push the image inside directory 'hpccpl-docker-local/hpcc-platform/ubuntu-22.04' , is that directory guaranteed to exist, and this new push path succeed?

Reply:
@jakesmith the hpccpl-docker-local at the beginning is the key to making it work properly. I tested the path Godji supplied earlier this evening and confirmed that it does work. I'll pass you the link on teams.